includes/class-wc-stripe-account.php

```diff
@@ -74,10 +74,9 @@ private function read_account_from_cache() {
 	private function cache_account() {
 		$expiration = 2 * HOUR_IN_SECONDS;
 
-		try {
-			// need call_user_func() as (  $this->stripe_api )::retrieve this syntax is not supported in php < 5.2
-			$account = call_user_func( [ $this->stripe_api, 'retrieve' ], 'account' );
-		} catch ( WC_Stripe_Exception $e ) {
+		// need call_user_func() as (  $this->stripe_api )::retrieve this syntax is not supported in php < 5.2
+		$account = call_user_func( [ $this->stripe_api, 'retrieve' ], 'account' );
```

Comment:
So `WC_Stripe_API::retrieve` doesn't actually throw a `WC_Stripe_Exception` or any exception for that matter, but it does return a `WP_Error` if the response is...also a `WP_Error`, so I updated this to handle this properly.